src/main/java/carpet/script/api/Auxiliary.java

```diff
@@ -1240,6 +1244,62 @@ else if (fdesc.type == FileArgument.Type.JSON)
             cc.host.issueDeprecation("enable_hidden_dimensions in 1.18.2 and 1.19+");
             return Value.NULL;
         });
+
+        expression.addUnaryFunction("item_nutrition", v-> NumericValue.of(NBTSerializableValue.parseItem(v.getString(), CarpetServer.minecraft_server.registryAccess()).getItem().getFoodProperties().getNutrition()));
+        expression.addUnaryFunction("item_saturation", v-> NumericValue.of(NBTSerializableValue.parseItem(v.getString(), CarpetServer.minecraft_server.registryAccess()).getItem().getFoodProperties().getSaturationModifier()));
+
+        expression.addContextFunction("food_properties", -1, (c, t, lv)->{
+            if(lv.size()==0) throw new InternalExpressionException("'food_properties' requires at least an item name");
+            if(lv.size()>2) throw new InternalExpressionException("'food_properties' takes at most an item name and a property");
+
+            Item item = NBTSerializableValue.parseItem(lv.get(0).getString(), CarpetServer.minecraft_server.registryAccess()).getItem();
```

Comment:
I didn't want to use `CarpetServer.minecraft_server.registryAccess()` here, but the function requires a `RegistryAccess` to be passed in.
Is there some better way?

Reply:
CarpetContext has it.